Assets/Scripts/Players/Player.cs

```diff
@@ -1,14 +1,14 @@
 using System;
 using Players.Behaviour;
 using UnityEngine;
-using UnityEngine.Serialization;
 
 namespace Players {
     public class Player : MonoBehaviour {
         [SerializeField] internal SpriteRenderer sprite;
         [SerializeField] internal Animator animator;
         [SerializeField] internal Rigidbody2D rb;
-        [SerializeField] internal LayerMask ground;
+        [SerializeField] internal LayerMask ground;      // includes semisolid platforms
+        [SerializeField] internal LayerMask solidGround; // does not include semisolid platforms
```

Comment:
would prefer something like fields `ground` and `semisolid`, and do `ground | semisolid` where both are needed